x-pack/plugins/apm/public/components/app/service_overview/service_overview_transactions_table/index.tsx

```diff
@@ -64,9 +66,37 @@ const StyledTransactionDetailLink = styled(TransactionDetailLink)`
   ${truncate('100%')}
 `;
 
-export function ServiceOverviewTransactionsTable(props: Props) {
-  const { serviceName } = props;
+function getLatencyColumnName(latencyAggregationType?: LatencyAggregationType) {
```

Comment:
Perhaps

```suggestion
function getLatencyAggregationTypeLabel(latencyAggregationType?: LatencyAggregationType) {
```

To be consistent with `latencyAggregationType` (even though it's very long...)